docs/migrate_from_ibmq.rst

```diff
@@ -0,0 +1,51 @@
+#############################################
+Migration guide from ``qiskit-ibmq-provider``
+#############################################
+
+Introduction
+============
+
+The classes related with Qiskit Runtime that used to be included in ``qiskit-ibmq-provider`` are now part of ``qiskit-ibm-runtime``. Before, the provider used to populate the ``qiskit.providers.ibmq.runtime`` namespace with objects for Qiskit Runtime that now live in the ``qiskit_ibm_runtime`` module.
+
+Changes in Class name and location
+==================================
+
+The module from which the classes are imported changed. Below is a table of example access patterns in ``qiskit.providers.ibmq.runtime`` and the new form with ``qiskit_ibm_runtime``:
+
+.. list-table:: Migrate from ``qiskit.providers.ibmq.runtime`` in ``qiskit-ibmq-provider`` to ``qiskit-ibm-runtime`` 
+   :header-rows: 1
+
+   * - class in ``qiskit-ibmq-provider``
+     - class in ``qiskit-ibm-runtime``
+     - Notes
+   * - ``qiskit.providers.ibmq.runtime.IBMRuntimeService``
+     - :class:`qiskit_ibm_runtime.QiskitRuntimeService`
+     - ``IBMRuntimeService`` class was removed from ``qiskit_ibm_runtime 0.6.0`` and replaced by :class:`~QiskitRuntimeService`.
```

Comment:
Not super familiar with rst files but will :class:`~QiskitRuntimeService` render the link correctly?

Reply:
If you use `intersphinx` extension it should work: https://qiskit.github.io/qiskit_sphinx_theme/sphinx_guide/how_to_cross-reference.html#pages-from-other-documentation-projects

Reply:
No worries.. I will go the traditional way.